ui/Screen/SendFunds.svelte

```diff
@@ -0,0 +1,14 @@
+<script>
+  import { pop } from "svelte-spa-router";
+  import { get } from "svelte/store";
+
+  import { fromStore, toStore, amountStore } from "../src/transfer.ts";
+
+  const fromAddress = get(fromStore);
```

Comment:
The `get` helper is really only supposed to be used in odd cases like testing or when we can't subscribe to a store (see the [svelte docs on `get`](https://svelte.dev/docs#get)). For svelte components, you should subscribe to stores like this: `$fromStore`.

Reply:
@sarahscott Thank you for pointing it out! I tried to address it in f32a9d5, let me know if it needs further change

Reply:
Just pushed a commit with what I think is a svelty way to do the reactivity. Let me know if you have any questions!